userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -617,6 +617,8 @@ The following section explains how to install the database:
 
 ```
 
+{{% readfile file="installation.md" code="true" lang="c++" %}}
+
```

Comment:
Why was this added? See the preview at https://deploy-preview-1413--docsydocs.netlify.app/docs/adding-content/shortcodes/#reuse-documentation, an error is reported:

> <img width="853" alt="image" src="https://user-images.githubusercontent.com/4140793/218506733-68f69fc1-9d32-4340-a8c7-57d8a66ba598.png">

If there is a change needed in this page, consider submitted it through another PR since this one is meant to be about config file renames.